pkg/controllers/csi/metadata/path_resolver.go

```diff
@@ -40,7 +48,7 @@ func (pr PathResolver) LatestAgentBinaryForDynaKube(dynakubeName string) string
 }
 
 func (pr PathResolver) AgentTempUnzipRootDir() string {
-	return filepath.Join(pr.RootDir, "tmp_zip")
+	return pr.Base("tmp_zip")
```

Comment:
wdyt if we move it to constants?

Reply:
I would leave it as is for now, this supposed to be a "quick" bugfix 😅